### PR TITLE
New GreenGrass CLI examples

### PR DESCRIPTION
**Pull request by bisdavid**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/examples/greengrass/create-logger-definition-version.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+**To create a logger definition version**
+
+The following ``create-logger-definition-version`` example creates a logger definition version and associates it with a logger definition. The version defines four logging configurations: 1) system component logs on the file system of the core device, 2) user-defined Lambda function logs on the file system of the core device, 3) system component logs in Amazon CloudWatch Logs, and 4) user-defined Lambda function logs in Amazon CloudWatch Logs. Note: For CloudWatch Logs integration, your group role must grant appropriate permissions. ::
+
+    aws greengrass create-logger-definition-version \
+        --logger-definition-id "a454b62a-5d56-4ca9-bdc4-8254e1662cb0" \
+        --loggers "[{"Id":"1","Component":"GreengrassSystem","Level":"ERROR","Space":10240,"Type":"FileSystem"},{"Id":"2","Component":"Lambda","Level":"INFO","Space":10240,"Type":"FileSystem"},{"Id":"3","Component":"GreengrassSystem","Level":"WARN","Type":"AWSCloudWatch"},{"Id":"4","Component":"Lambda","Level":"INFO","Type":"AWSCloudWatch"}]"
```
Comment (kyleknap) [inline awscli/examples/greengrass/create-logger-definition-version.rst]: Should the outter double quotes being singe quotes instead? Without it, I think the inner double quotes would need to be escaped.

File: awscli/examples/greengrass/create-logger-definition.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+**To create a logger definition**
+
+The following ``create-logger-definition`` example creates a logger definition that contains an initial logger definition version. The initial version defines three logging configurations: 1) system component logs on the file system of the core device, 2) user-defined Lambda function logs on the file system of the core device, and 3) user-defined Lambda function logs in Amazon CloudWatch Logs. Note: For CloudWatch Logs integration, your group role must grant appropriate permissions. ::
+
+    aws greengrass create-logger-definition \
+        --name "LoggingConfigs" \
+        --initial-version "{"Loggers":[{"Id":"1","Component":"GreengrassSystem","Level":"ERROR","Space":10240,"Type":"FileSystem"},{"Id":"2","Component":"Lambda","Level":"INFO","Space":10240,"Type":"FileSystem"},{"Id":"3","Component":"Lambda","Level":"INFO","Type":"AWSCloudWatch"}]}"
```
Comment (kyleknap) [inline awscli/examples/greengrass/create-logger-definition.rst]: Same question here about the quoting

File: awscli/examples/greengrass/create-logger-definition-version.rst (hunk 1 above)
Comment (bisdavid) [reply awscli/examples/greengrass/create-logger-definition-version.rst]: You're correct, of course.  :-) 

I'll fix and resubmit.

Comment (bisdavid) [reply awscli/examples/greengrass/create-logger-definition-version.rst]: Fixed both instances in the create-logger-definition and create-logger-definition-version command.

File: awscli/examples/greengrass/create-logger-definition.rst (hunk 2 above)
Comment (bisdavid) [reply awscli/examples/greengrass/create-logger-definition.rst]: Fixed both.

**Pull request merged**